### PR TITLE
Ref guide overall version improvement

### PR DESCRIPTION
**Pull request by HoustonPutman**
- Moved the version selector up to the title.
- Added a perma-link button to get the permanent link for a page.
- Made the page-version selector orange, to make it more visible.

Looks like this:

Screen Shot 2022-05-12 at 6 49 47 PM

### CODE REVIEW

**Comment by madrob** [comment]
Are there two version selectors now?

**Comment by madrob** [comment]
Where do the new files come from? Copied from the same Antora default templates?

**Comment by HoustonPutman** [comment]
> Are there two version selectors now?

there always have been. I just made each more prominent.



> Where do the new files come from? Copied from the same Antora default templates?

yes, (almost) all files in ui-src are copied from the antora template and modified. I have made sure that all changes, additions and removals are noted, so that we can upgrade these files in the future.

File: solr/solr-ref-guide/ui-src/js/07-copy-perma-link.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    span.appendChild(toolbox)
+    if (copy) copy.addEventListener('click', writeToClipboard.bind(copy, link))
+  })
+
+  function writeToClipboard (link) {
+    var text = link.replace(TRAILING_SPACE_RX, '')
+    window.navigator.clipboard.writeText(text).then(
+      function () {
+        this.classList.add('clicked')
+        this.offsetHeight // eslint-disable-line no-unused-expressions
```
[the remark this answers is not in the document]
Comment (HoustonPutman) [reply solr/solr-ref-guide/ui-src/js/07-copy-perma-link.js:45]: @sonatype-lift ignore

**Comment by HoustonPutman** [comment]
You can build this yourself and test it out with:
```
gradle buildOfficialSite -Prefguide.official.includePrereleaseVersions=true -Prefguide.buildLocalUI=true
```

New look of link:

Screen Shot 2022-05-13 at 6 16 36 PM

Hover:

Screen Shot 2022-05-13 at 6 17 57 PM

**Pull request merged**